pkg/builtin/manifest.yml

```diff
@@ -1918,3 +1918,15 @@ extensions:
               title: Layer
               type: ref
               ui: layer
+  - id: timeline
+    type: widget
+    name: Timeline
+    description: Operate timeline for earth.
+    singleOnly: false
```

Comment:
This should be true, shouldn't it? Do we actually want multiple timelines??

Reply:
This was fixed by [fix: enable singleOnly flag](https://github.com/reearth/reearth-backend/pull/161/commits/27996b0eaeebb67f482db8e2b9c26f859cf83479)